api/models/fixtures/web_external_internal.json

```diff
@@ -165,14 +178,32 @@
         ],
         "Listeners": [
           {
-            "InstancePort": {
-              "Ref": "WebPort80Host"
-            },
-            "InstanceProtocol": "TCP",
-            "LoadBalancerPort": {
-              "Ref": "WebPort80Balancer"
-            },
-            "Protocol": "TCP"
+            "Fn::If": [
+              "BlankWebPort80Certificate",
+              {
+                "InstancePort": {
+                  "Ref": "WebPort80Host"
+                },
+                "InstanceProtocol": "TCP",
+                "LoadBalancerPort": {
+                  "Ref": "WebPort80Balancer"
+                },
+                "Protocol": "TCP"
+              },
+              {
+                "InstancePort": {
+                  "Ref": "WebPort80Host"
+                },
+                "InstanceProtocol": "TCP",
```

Comment:
Looks like it's being hardcoded to unencrypted container port right now which means rack and grid themselves wouldn't work with this.